examples/s3/bucketpolicy-json.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: s3.aws.crossplane.io/v1alpha2
+kind: BucketPolicy
+metadata:
+  name: bucketpolicy-public
+spec:
+  forProvider:
+    region: us-west-1
+    bucketNameRef:
+      name: test-bucket
+    jsonBody: '{"Statement":[{"Action":"s3:ListBucket","Effect":"Allow","Principal":"*","Resource":"arn:aws:s3:::test-bucket"}],"Version":"2012-10-17"}'
```

Comment:
Example should be updated to reflect newer `rawPolicy` field.

Reply:
Also nit, but I would prefer the example `rawPolicy` have more readable format than compact json, like

```yaml
rawPolicy: |
  {
   "Statement":[
      {
         "Action":"s3:ListBucket",
         "Effect":"Allow",
         "Principal":"*",
         "Resource":"arn:aws:s3:::test-bucket"
      }
   ],
   "Version":"2012-10-17"

```

Reply:
Thanks for the review! These changes have been made and pushed :)